code_schemes/s01e01.json

```diff
@@ -89,6 +89,62 @@
       "StringValue": "kenya_update",
       "VisibleInCoda": true
     },
+    {
+      "CodeID": "code-53399b8d",
+      "CodeType": "Normal",
+      "DisplayText": "lack of individual resources",
```

Comment:
This is "(individual)" in the thematic document (not sure why). It could have parentheses in the displaytext too if it helps